cmd/config/set.go

```diff
@@ -46,13 +46,9 @@ Can set the below settings:
 			err = entry.Set(key, value)
 			if err != nil {
 				if isInvalidKeyError(err) {
-					err := cmd.Help()
-					if err != nil {
-						return err
-					}
-				} else {
-					return err
+					cmd.Help()
```

Comment:
I think some IDEs classify this as an warning (discarding error value without any notice), so I suggest to clearly drop it. 
```suggestion
					_ = cmd.Help()
```